release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3326,6 +3326,40 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+[source,terminal]
+----
+$ oc adm release info 4.14.33 --pullspecs
+----
+
+[id="ocp-4-14-33-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, nodes that were booted using 4.1 and 4.2 bootimages for {product-title} got stuck during provisioning because the `machine-config-daemon-firstboot.service` had incompatible machine-config-daemon binary code. With this release, binary has been updated and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36776[*OCPBUGS-36776*])
+
+* Previously, a change of dependency targets was introduced in {product-title} 4.14 that prevented disconnected ARO installs from scaling up new nodes after they upgraded to affected versions. With this release, disconnected ARO installs can scale up new nodes after upgrading to {product-title} 4.14. (link:https://issues.redhat.com/browse/OCPBUGS-36593[*OCPBUGS-36593*])
+
+* Previously, if a new deployment was performed at the OSTree level on a host that was identical to the current deployment but on a different stateroot, the OSTree saw them as equal. This behavior incorrectly prevented the boot loader from updating when `set-default` was invoked, as OSTree did not recognize the two stateroots as a differentiation factor for deployments. With this release, OSTree's logic has been modified to consider the stateroots and allows OSTree to properly set the default deployment to a new deployment with different stateroots. (link:https://issues.redhat.com/browse/OCPBUGS-36437[*OCPBUGS-36437*])
+
+* Previously, the `HighOverallControlPlaneCPU` alert triggered warnings based on criteria for multi-node clusters with high availability. As a result, misleading alerts were triggered in {sno} clusters because the configuration did not match the environment criteria. This update refines the alert logic to use {sno}-specific queries and thresholds and account for workload partitioning settings. As a result, CPU utilization alerts in {sno} clusters are accurate and relevant to single-node configurations. (link:https://issues.redhat.com/browse/OCPBUGS-31354[*OCPBUGS-31354*])
+
+* Previously, the DNS Operator did not verify that a cluster had ready nodes with available CPU in at least two availability zones, and the DNS daemon set did not use surge for rolling updates. As a result, clusters in which all nodes were in the same availability zone would repeatedly emit `TopologyAwareHintsDisabled` events for the cluster DNS service. With this release, the `TopologyAwareHintsDisabled` events are no longer emitted on clusters that do not have nodes in multiple availability zones and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-5943[*OCPBUGS-5943*])
+
+[id="ocp-4-14-33-updating_{context}"]
+==== Updating
+
+To update an existing {product-title} 4.14 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Comment:
```suggestion
To update an {product-title} 4.14 cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Reply:
Good catch!

Reply:
This language has been in the release notes for a long time so I'm reluctant to change it.